app/assets/stylesheets/application.scss

```diff
@@ -5,18 +5,13 @@ $govuk-new-link-styles: true;
 @import "govuk_publishing_components/govuk_frontend_support";
 @import "govuk_publishing_components/component_support";
```

Comment:
Good question. I think we might need to look into this - originally I split the govuk-frontend imports into these two files because `govuk_frontend_support` adds important mixins but no asset weight, whereas `component_support` adds all the rest, including asset weight.

However since static is now including both of them (not sure exactly when that happened, as this file was renamed from `static.scss` in [2021](https://github.com/alphagov/static/blob/fc4f99a343c837583d367485516c8db2ae060019/app/assets/stylesheets/application.scss) - and I think at that point we weren't including any components in static), the weight question is a bit null and void.

If we're just including everything in static then maybe we should have both files in static, then only have `govuk_frontend_support` in all the apps (so they can access the mixins).

Reply:
Rather than block this PR, can we agree to look into this across all apps and leave this PR as it is for now?

Reply:
The change was done in around April 2021 - there's notes from the frontenders meeting on the 13 April 2021 where we discussed stopping the used of Sass's `@extend` as it added too much to the CSS. Removing `component_support` from the application and putting in Static's stylesheet meant that `@extend` wasn't an option, but using the `govuk-` classes were thanks to CSS's glorious global namespace.

Since `component_support` already included in Static, it doesn't need to be included here - it's serving the same code twice to users. A quick check shows that `frontend`, `finder-frontend`, `collections`, `whitehall`, `service-manual-frontend`, and `info-frontend` already don't include component support with no ill effect.

I don't want to block this, but I also don't understand why this needs further investigation when it's already been successfully done in other applications, it was discussed with the community, and it fits within the remit of this pull request.